### DISCUSSION TITLE
启动项目后终端提示这个警告 ：WARN  [unocss] unmatched utility "lt-lg:__container" in shortcut "container"

### DISCUSSION THREAD

**Discussion by songpeng154** [unocss/unocss#4216] [Q&A]
不明白为什么要提示这个警告

**Comment by zyyv** [comment]
What's your usage about `container`, would you like to paste it?

**Reply by songpeng154** [reply]
> 您对 有何用途`container`，要粘贴吗？

Resolved, thank you for your reply
